hummingbot/connector/derivative/binance_perpetual/constants.py

```diff
@@ -41,16 +43,26 @@
 ACCOUNT_INFO_URL = "/account"
 POSITION_INFORMATION_URL = "/positionRisk"
 
-
 # Private API Endpoints
 BINANCE_USER_STREAM_ENDPOINT = "/listenKey"
 
+# Order Statuses
+ORDER_STATE = {
+    # "PENDING_NEW": OrderState.PENDING_CREATE,
+    "NEW": OrderState.OPEN,
+    "FILLED": OrderState.FILLED,
+    "PARTIALLY_FILLED": OrderState.PARTIALLY_FILLED,
+    "CANCELED": OrderState.CANCELLED,
+    # "FAILED": OrderState.FAILED,
+}
+
```

Comment:
According to the official documentation (https://binance-docs.github.io/apidocs/futures/en/#public-endpoints-info) all possible order status alternatives are:
NEW
PARTIALLY_FILLED
FILLED
CANCELED
REJECTED
EXPIRED

Not all of them are included in the dictionary, and that could cause troubles when parsing messages or events.